### PR TITLE
fix: added 'exact', fixed bundles

### PR DESCRIPTION
**Pull request by maria-j-k**

### CODE REVIEW

File: backend/app/consts.py [hunk 1]
```diff
@@ -62,6 +63,7 @@ class PanelId(str, Enum):
     Collection.SERVICE: PanelId.SERVICES,
     Collection.TRAINING: PanelId.TRAININGS,
     Collection.SOFTWARE: PanelId.SOFTWARE,
+    Collection.BUNDLE: PanelId.BUNDLE,
```
Comment (Michal-Kolomanski) [inline backend/app/consts.py:66]: It is not the way. We should not send recommendation requests at all for bundles (like for providers or guidelines for example)

Comment (Michal-Kolomanski) [reply backend/app/consts.py:66]: Not relevant for now

**Review by Michal-Kolomanski — APPROVED** [review]

**Comment by Marcelinna** [comment]
@maria-j-k @Michal-Kolomanski I don't see recommendations for training, it is a bug?

**Comment by Marcelinna** [comment]
lack of Pin the Marketplace project button for Datasurce
image

**Review by michal-szostak — APPROVED** [review]

**Review by Marcelinna — APPROVED** [review]

**Review by eldenkat — APPROVED** [review]

**Pull request merged**